Libraries/PeriphDrivers/Source/AFE/afe.c

```diff
@@ -59,14 +75,66 @@
 // AFE Trim Storage Defines
 //#define DUMP_TRIM_DATA
 
-#if (TARGET != MAX32675 || TARGET_NUM == 32675)
+//
+// This timeout prevents infinite loop if AFE is not responsive via SPI
+//
+//  Each read from AFE is 8 bits, unless CRC is enabled.
+//  maximum of 4 reads.
+//      1/AFE_SPI_BAUD * 4 * 8 should be a reasonable timeout for this.
+//      1/100000 = 10us * 4 * 8 = 320us round up for safety: 400us
+//
+//  Since this is just for catastrophic lockup, lets double it.
+//
+#if (AFE_SPI_BAUD != 100000)
+#warning "Recalculate MXC_AFE_SPI_READ_TIMEOUT, since baud rate was modified.\n"
+#endif
+
+#define MXC_AFE_SPI_READ_TIMEOUT USEC(800)
+
+//
+// Initial AFE RESET time, up to 10 milliseconds
+//
+#define MXC_AFE_SPI_RESET_TIMEOUT MSEC(10)
+
+#if (TARGET_NUM == 32675)
 #define AFE_TRIM_OTP_OFFSET_LOW 0x280
 #define AFE_TRIM_OTP_OFFSET_HIGH 0x288
-#elif (TARGET != MAX32680 || TARGET_NUM == 32680)
+
+#define AFE_SPI_MISO_GPIO_PORT MXC_GPIO0
+#define AFE_SPI_MISO_GPIO_PIN MXC_GPIO_PIN_2
+
+#define AFE_SPI_MOSI_GPIO_PORT MXC_GPIO0
+#define AFE_SPI_MOSI_GPIO_PIN MXC_GPIO_PIN_3
+
+#define AFE_SPI_SCK_GPIO_PORT MXC_GPIO0
+#define AFE_SPI_SCK_GPIO_PIN MXC_GPIO_PIN_4
+
+#define AFE_SPI_SSEL_GPIO_PORT MXC_GPIO0
+#define AFE_SPI_SSEL_GPIO_PIN MXC_GPIO_PIN_5
+
+#elif (TARGET_NUM == 32680)
 #define AFE_TRIM_OTP_OFFSET_LOW 0x0E10
 #define AFE_TRIM_OTP_OFFSET_HIGH 0x0E18
+
+#define AFE_SPI_MISO_GPIO_PORT MXC_GPIO0
+#define AFE_SPI_MISO_GPIO_PIN MXC_GPIO_PIN_20
+
+#define AFE_SPI_MOSI_GPIO_PORT MXC_GPIO0
+#define AFE_SPI_MOSI_GPIO_PIN MXC_GPIO_PIN_21
+
+#define AFE_SPI_SCK_GPIO_PORT MXC_GPIO0
+#define AFE_SPI_SCK_GPIO_PIN MXC_GPIO_PIN_22
+
+#define AFE_SPI_SSEL_GPIO_PORT MXC_GPIO0
+#define AFE_SPI_SSEL_GPIO_PIN MXC_GPIO_PIN_23
 #endif
 
+//
+// Register defines for new AFE reset on ME16A-0D
+//
+#define MXC_F_GCR_RST1_AFE_POS 25 /**< RST1_AFE Position */
+#define MXC_F_GCR_RST1_AFE ((uint32_t)(0x1UL << MXC_F_GCR_RST1_AFE_POS)) /**< RST1_AFE Mask */
```

Comment:
The correct location for these definitions seems gcr_regs.h file.
All GCR related definition in that file. To centralize thing it would be better if you move it to there.

https://github.com/Analog-Devices-MSDK/msdk/blob/main/Libraries/CMSIS/Device/Maxim/MAX32675/Include/gcr_regs.h

https://github.com/Analog-Devices-MSDK/msdk/blob/main/Libraries/CMSIS/Device/Maxim/MAX32680/Include/gcr_regs.h